### PR TITLE
Nan handling metadata

### PR DESCRIPTION
**Pull request by Mupico**
text metadata now able to work with nan values

### CODE REVIEW

**Comment by Mupico** [comment]
math.isnan is only for float values. since I work with strings or dicts usually, calling math.isnan throws an error

File: tests/precalculations/test_text_metadata.py [hunk 1]
```diff
@@ -11,6 +11,27 @@
 import tests.test_data as td
 
 
+import math
+
+
+class NumericAssertions:
+    def assertIsNaN(self, value, msg=None):
```
Comment (velramiir) [inline tests/precalculations/test_text_metadata.py]: use snake_case for functions

File: tests/precalculations/test_text_metadata.py [hunk 2]
```diff
@@ -11,6 +11,27 @@
...
+        standardMsg = "%s is not NaN" % str(value)
+        try:
+            if not math.isnan(value):
+                self.fail(self._formatMessage(msg, standardMsg))
+        except:
+            self.fail(self._formatMessage(msg, standardMsg))
+
+    def assertIsNotNaN(self, value, msg=None):
```
Comment (velramiir) [inline tests/precalculations/test_text_metadata.py]: same here

**Comment by Mupico** [comment]
pandas.isna does work with strings, but not on arrays

**Pull request merged**